File: problems/problem_2/fullboilerplate.java
import java.util.*;
public class Main {
    // {{USER_CODE}}
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        if (sc.hasNext()) {
            String s = sc.next();
            boolean res = isPalindrome(s);
            System.out.println(res ? "true" : "false");
        }
    }
}

File: problems/problem_2/fullboilerplate.cpp
#include <bits/stdc++.h>
using namespace std;
// {{USER_CODE}}
int main() {
    string s;
    if (cin >> s) {
        bool res = isPalindrome(s);
        cout << (res ? "true" : "false") << endl;
    }
    return 0;
}